variables.tf

```diff
@@ -382,3 +382,26 @@ variable "restore_to_point_in_time" {
   description = "An object specifying the restore point in time for the DB instance to restore from. Only used when `snapshot_identifier` is not provided."
   default     = null
 }
+
+variable "use_reserved_instances" {
```

Comment:
Cloud posse convention (and ours too!): Use `_enabled` for `bool` type variables. [See here](https://docs.cloudposse.com/best-practices/terraform/#use-positive-variable-names-to-avoid-double-negatives).